### PR TITLE
Fix QuickLogic install job

### PR DESCRIPTION
**Pull request by lpawelcz**
This PR moves QuickLogic installed toolchain tests to a separate job in order to fix package generation for QL. It also fixes some inconsistencies in locations of utility scripts. This PR is Cross-dependent with: https://github.com/chipsalliance/f4pga/pull/587

### CODE REVIEW

File: .github/workflows/Tests.yml
Comment (umarcor) [inline .github/workflows/Tests.yml:70]: Please use `name: install-${{ matrix.install }}`. There is another job named xc7 in the Test matrix, and it might be confusing.

Comment (lpawelcz) [reply .github/workflows/Tests.yml:70]: I edited the whole job name inside `install` matrix. Is that ok?

**Comment by lpawelcz** [comment]
Also cross-dependent with: https://github.com/chipsalliance/f4pga-examples/pull/338

**Review by umarcor — COMMENTED** [review]
LGTM!

It feels that `install_ql.sh` and `install_xc7.sh` might be combined into a single script that accepts an argument to select the family. Nonetheless, that's out of scope of this PR. We can handle it afterwards.

On the other hand, I don't see where is `ql.sh` used at the moment. Again, not relevant to this PR, but we should have a look at it later.

I won't mark this as approved now, because we need to wait until CI is green and then remove the temporary commit to use the f4pga branch. Other than that, very nice work. Congrats!

**Comment by lpawelcz** [comment]
@umarcor `ql.sh` is used in this test: https://github.com/SymbiFlow/f4pga-arch-defs/blob/a11cc174344c83bfb2cc8db77b5ba3abf9c17f82/.github/workflows/Tests.yml#L28
It runs a series of test designs from internal cmake infrastructure while `install_ql` installs the toolchain and uses it directly.

**Comment by lpawelcz** [comment]
CI went green with temporary reference to custom `f4pga` branch.
CI run: https://github.com/SymbiFlow/f4pga-arch-defs/actions/runs/2753793137
temporary commit: https://github.com/SymbiFlow/f4pga-arch-defs/pull/2821/commits/a11cc174344c83bfb2cc8db77b5ba3abf9c17f82

@umarcor I deleted this commit so we can merge this PR.

**Review by tmichalak — APPROVED** [review]
LGTM

**Pull request merged**
